Deploy-with-tls.md

```diff
@@ -0,0 +1,84 @@
+## Install kubelogin with TLS and Unique Password
+It is assumed you have completed the steps in the main [README](./README.md) file up to point where you deployed installed Kustomize. Follow the steps below to deploy kubeflow and provide access to it with TLS. Please note that a self-signed certificate is used for demonstration purposes. Do not use self signed certs for production workloads. You can easily swap this self-signed cert with your CA certificate for your usecase.
+
+    > :warning:For this deployment, we will be using a simple method for authenticating to Kubeflow. For more advanced usecases, please configure your deployment to use Azure AD.
+1. The first step is to generate a new Hash/Password combination using bycrypt. There are many ways of doing this, eg by generating it [using python](https://www.geeksforgeeks.org/hashing-passwords-in-python-with-bcrypt/). For simplicity we will be using coderstool's [Bycrypt Hash Generator](https://www.coderstool.com/bcrypt-hash-generator) for testing purposes. Do not do this for production workloads. In the plane text field, enter a password for your first user, then click on the "Generate Hash" button. You can generate multiple if you have multiple users.
+    ![Generate password](./media/brypt-password-generation.png)
+1. Head to the tls-manifest/manifests/common/dex/base/config-map.yaml file and update the hash value there (around line 22) with the hash you just generated. You can also change the email address, username and userid. In addition, you can setup multiple users by adding more users to the array.
+1. Update your auth.md file with the new email address and password (plain text password not hash) or store the secrets in a more secure way
+1. Copy the contents of this newly updated manifests folder to the kubeflow manifests folder. This will update the files so the deployment includes your config changes.
+    ```bash
+    cp -r tls-manifest/manifests/ manifests/
+    ```
+1. cd to the manifests folder and install kubeflow
+    ```bash
+    cd manifests
+    ```
+    Install all of the components via a single command
+    
+    ```bash
+    while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+    ```  
+1. Once the command has completed, check the pods are ready
+    
+    ```bash
+    kubectl get pods -n cert-manager
+    kubectl get pods -n istio-system
+    kubectl get pods -n auth
+    kubectl get pods -n knative-eventing
+    kubectl get pods -n knative-serving
+    kubectl get pods -n kubeflow
+    kubectl get pods -n kubeflow-user-example-com
+    ```
+1. Restart dex to ensure everything is working as expected
```

Comment:
We should stress that it is critical to restart dex to use the new password configuration. Otherwise the default password `12341234` will be used from the time the Service is exposed via `LoadBalancer` until the time this command is run.